docs(evm): add integration requirements for multisig accounts (#209)
axelarnetwork/axelar-utils-solidity: contracts/gateway/INTEGRATION.md

The upgrade mechanism is expected to vary for non-EVM chains (some chains support contract upgrades natively).

### Multisig

A multisig might be used for certain privileged roles in the smart contracts, e.g. the `operator` account in the `AxelarAmplifierGateway` and `AxelarServiceGovernance`, the `owner` of the `AxelarAmplifierGateway` (temporarily, until the transfer to Governance), and the `owner` of some less critical contracts like the `AxelarGasService`.

> For EVM chains, a custom [Multisig contract](../governance/Multisig.sol) is being used. However, another approach might make sense for other chain integrations. If the chain supports native multisig accounts, that might be preferrable. If it doesn't or it's capability is limited, we recommend using the preferred Multisig contract that's standard in the ecosystem (e.g. for EVM chains, [Safe](https://github.com/safe-global/safe-smart-account) is the most popular multisig). If this isn't available, then a custom contract needs to be written.

Author: ```suggestion
For EVM chains, a custom [Multisig contract](../governance/Multisig.sol) is being used. However, another approach might make sense for other chain integrations. If the chain supports native multisig accounts, that might be preferrable. If it doesn't or it's capability is limited, we recommend using the preferred Multisig contract that's standard in the ecosystem (e.g. for EVM chains, [Safe](https://github.com/safe-global/safe-smart-account) is the most popular multisig). If this isn't available, then a custom contract needs to be written.

For contracts that live on Axelar Amplifier itself, the Axelar governance can be the owner. For other privileged roles, the native multisig accounts on Axelar can be used if needed.
```

Requirements:

> 1. Supports a `k`-of-`n` signer set. `n` is unlikely to be greater than 10.

Author: ```suggestion
1. Supports a `k`-of-`n` signer set. `n` is not expected to be greater than 10.
```

The upgrade mechanism is expected to vary for non-EVM chains (some chains support contract upgrades natively).

### Multisig

A multisig might be used for certain privileged roles in the smart contracts, e.g. the `operator` account in the `AxelarAmplifierGateway` and `AxelarServiceGovernance`, the `owner` of the `AxelarAmplifierGateway` (temporarily, until the transfer to Governance), and the `owner` of some less critical contracts like the `AxelarGasService`.

For EVM chains, a custom [Multisig contract](../governance/Multisig.sol) is being used. However, another approach might make sense for other chain integrations. If the chain supports native multisig accounts, that might be preferrable. If it doesn't or it's capability is limited, we recommend using the preferred Multisig contract that's standard in the ecosystem (e.g. for EVM chains, [Safe](https://github.com/safe-global/safe-smart-account) is the most popular multisig). If this isn't available, then a custom contract needs to be written.

> For contracts that live on Axelar Amplifier itself, the Axelar governance can be the owner. For other privileged roles, the native multisig accounts on Axelar can be used if needed.

Author: ```suggestion
For CosmWasm contracts that live on Axelar itself, the Axelar governance can be the owner. For other privileged roles, the native multisig accounts on Axelar can be used if needed.
```
